packages/image-utils/src/Image.ts

```diff
@@ -212,3 +214,12 @@ export async function compositeImagesAsync({
     .composite([{ input: foreground, left: x, top: y }])
     .toBuffer();
 }
+
+export async function isValidPng(src: string) {
```

Comment:
Alternative method signature:

```js
isValidPng(src: string): { isValid: boolean, errorMessage?: string }
```
or
```js
isValidPng(src: string): { isValid: boolean, error?: Error }
```
or even
```js
isValidPng(src: string): { isValid: boolean, errorCode?: string, errorMessage?: string, error?: Error }
```

For the test cases I added I've seen two errors:
1. `ENOENT: no such file or directory, open 'random/path'`
2. `Invalid file signature`

So I could either map these to error codes or pass them down as is, so we could show a more informative error message ("check <path> is a valid `.png`" vs "file at <path> does not exist".

Not sure where the user-facing error message should live. Thoughts?